sdk/resourcemanager/azure-resourcemanager-compute/src/test/resources/VirtualMachineSizeTypes.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.compute.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for VirtualMachineSizeTypes. */
+public final class VirtualMachineSizeTypes extends ExpandableStringEnum<VirtualMachineSizeTypes> {
+TYPES_PLACE_HOLDER
```

Comment:
This doesn't appear as if it would compile?

Reply:
It's placed under `../src/test/resources/..` as template file. We prepared one test method to pull all the values and generate/update the enum class by this template.